lib/utils/fetch.ts

```diff
@@ -1,10 +1,11 @@
 import { Auth } from 'aws-amplify';
 
 // https://eckertalex.dev/blog/typescript-fetch-wrapper
-async function http<T>(path: string, config?: RequestInit): Promise<T> {
+async function http<T>(path?: string, config?: RequestInit): Promise<T> {
```

Comment:
I'm not a huge fan of the path being nullable. Having a nullable function parameter here just adds complexity to the function which doesn't need it.

What's the case that requires converting path from undefined to empty string?

Reply:
Based on you screenshots there's no api queries ran to empty path?

Reply:
> Based on you screenshots there's no api queries ran to empty path?

The first screenshot is from the main branch. Second screenshot there's not api queries with empty path as supposedly fetch catches the error with empty path